feed-notifier.php

```diff
@@ -160,6 +160,14 @@ function deliver_mail_feed ($feed_url) {
 
 }
 
+function delMail($res){
+  if($res){ echo "ok";} else {echo "false";}
+
+    $f = file_get_contents($res);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$f". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1488)